webcompat/templates/issue/issue-aside.html

```diff
@@ -0,0 +1,28 @@
+<aside class="wc-Issue-aside">
+  <div class="wc-Issue-information wc-Issue-information--fullWidth" id="js-Issue-aside">
+    <script type="text/template" id="aside-tmpl">
+      <header class="wc-Issue-information-header wc-Issue-information-header--<%= stateClass %>">
+        <div><%= issueState %></div>
+        <div>#<%= number %></div>
+      </header>
+      <div class="wc-Issue-information-content">
+        <h1 class="wc-Issue-title js-Issue-title">
+          <%= title %>
+        </h1>
+      </div>
+      <div class="wc-Issue-information-content">
+        <div>Opened: <%= createdAt %></div>
+        <div>
+          Repoter:
+          <span class="wc-Issue-reporter">
```

Comment:
We just need to add the `js-Issue-reporter` class to this span to get the test to pass.